packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -69,8 +80,12 @@ export default class SentryErrorLogger implements ErrorLogger {
                     iteratee: this.handleRewriteFrame,
                 }),
             ],
+            transport: 'fetch' in window ? makeFetchTransport : makeXHRTransport,
+            stackParser: defaultStackParser,
             ...config,
-        });
+        };
+
+        this.hub = createHub(clientOptions as BrowserClientOptions);
```

Comment:
Can we use a typeguard here and avoid the `as` keyword please?

Reply:
Yes good spot, I made some tweaks to the type so I no longer need to use `as`. 🙂